.travis/release.sh

```diff
@@ -23,6 +25,10 @@ sed "s~containerImage: docker.io/jaegertracing/jaeger-operator.*~containerImage:
 sed "s/name: jaeger-operator\.v.*/name: jaeger-operator.v${OPERATOR_VERSION}/gi" -i deploy/olm-catalog/jaeger.clusterserviceversion.yaml
 sed "s~image: jaegertracing/jaeger-operator.*~image: ${BUILD_IMAGE}~gi" -i deploy/olm-catalog/jaeger.clusterserviceversion.yaml
 
+sed "s/all-in-one:.*\"/all-in-one:${JAEGER_VERSION}\"/gi" -i deploy/olm-catalog/jaeger.clusterserviceversion.yaml
```

Comment:
Note: this may not be necessary depending what updates are done to the example in the csv - we may still want to show how images can be specified even with production strategy?